src/bundle/Controller/LocationController.php

```diff
@@ -56,6 +56,13 @@ public function actionsAction(
             if ($actionsForm->get('delete')->isClicked()) {
                 $uiLocationService->deleteLocations($locationIds);
             }
+
+            if ($actionsForm->get('add')->isClicked()) {
+                $uiLocationService->addLocation(
+                    $content->contentInfo,
+                    $actionsForm->get('parentLocationId')->getData()
+                );
+            }
```

Comment:
and btw, it also a good occasion to test the new forms in tabs feature, so you should:

1. rebase on master
1. make sure to redirect the `locations_tab` view below